src/neuroglancer/webgl/colormaps.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export function colormaps() {
```

Comment:
Could your bundling script instead just create a colormaps.glsl.js file that has a single default export of the code as a string (effectively what webpack is doing)?  Then we don't need to change anything else.

Reply:
I had problems with the file not having a .ts extension during the transformation step. I will see if I can get that to work, but the essence of the problem was that tsc wouldn't convert and copy the files. I may have missed a flag, so I will look to see if this can be fixed.

Reply:
Actually, on further thought, since all of the GLSL code in Neuroglancer is specified as a string constant with a .ts file, it seems reasonable to do the same here and avoid the need to deal with this .glsl file specially.  For consistency with webgl/shader_lib.ts, you could just have it be a string constant named glsl_colormaps, e.g. export const glsl_colormaps = ` ... `;

Reply:
Seems fair. I will take care of that.